server/openapi-spec.yaml

```diff
@@ -157,10 +347,50 @@ components:
             - public
         certified:
           type: boolean
-    Response:
+    FormResponse:
       type: object
       properties:
         id:
           type: string
         isolationPlanStatus:
           type: boolean
+    Login:
+      properties:
+        username:
+          type: string
+        password:
+          type: string
+    LoginResponse:
+      type: object
+      properties:
+        token:
+          type: string
+    Phac:
+      required:
+      - covid_id
+      properties:
+        covid_id:
+          type: string
+      additionalProperties:
+        anyOf:
+        - type: string
+        - type: number
+        - type: integer
+        - type: boolean
+        - type: array
+          items: {}
+        - type: object
+        nullable: false
+    PhacResponse:
+      type: object
+      properties:
+        successful:
+          type: object
```

Comment:
This is the only thing I could really find that seems to remain undefined / without an example